code/espurna/mcp23s17.h

```diff
@@ -0,0 +1,33 @@
+/*
+
+MCP23S17 MODULE
+
+Copyright (C) 2020-2020 by Eddi De Pieri <eddi at depieri dot com>
+Copyright (C) 2016-2017 by Xose Pérez <xose dot perez at gmail dot com>
+Copyright (C) 2016-2017 Plamen Kovandjiev <p.kovandiev@kmpelectronics.eu> & Dimitar Antonov <d.antonov@kmpelectronics.eu>
+
+*/
+
+#pragma once
+
+#include "espurna.h"
+
+#if MCP23S17_SUPPORT
+
+#include <SPI.h>
+
+// Inputs and outputs count.
+#define RELAY_COUNT  4
+#define OPTOIN_COUNT 4
```

Comment:
Pls use prefixes.
Should these be ifndef guarded so we can override values?

Reply:
done/moved into hardware.h